### PR TITLE
feat(core): switch GLWE primitives to the new noise distribution system

### PR DESCRIPTION
**Pull request by IceTDrinker**
same as the previous PR for LWE primitives

### CODE REVIEW

File: tfhe/src/core_crypto/commons/math/random/mod.rs [hunk 1]
```diff
@@ -113,16 +113,28 @@ impl<T: UnsignedInteger> DynamicDistribution<T> {
...
 
     pub const fn new_t_uniform(bound_log2: u32) -> Self {
         Self::TUniform(TUniform::new(bound_log2))
     }
+
+    #[track_caller]
+    pub fn gaussian_variance(&self) -> Variance {
+        match self {
+            Self::Gaussian(gaussian) => {
+                Variance(StandardDev::from_standard_dev(gaussian.std).get_variance())
```
Comment (mayeul-zama) [inline tfhe/src/core_crypto/commons/math/random/mod.rs:131]: Not introduced in this PR, but `get_value` could return a `Variance`
Same for other `DispersionParameter` methods

**Review by mayeul-zama — COMMENTED** [review]
The changes look fine to me
Is there any specific change to review carefully?

File: tfhe/src/core_crypto/commons/math/random/mod.rs (hunk 1 above)
Comment (IceTDrinker) [reply tfhe/src/core_crypto/commons/math/random/mod.rs:131]: not sure I understand ?

**Comment by IceTDrinker** [comment]
> The changes look fine to me Is there any specific change to review carefully?

Mainly that std dev are well translated to their new gaussian counterparts where necessary and that encryption function do what they are supposed to do

File: tfhe/src/core_crypto/commons/math/random/mod.rs (hunk 1 above)
Comment (IceTDrinker) [reply tfhe/src/core_crypto/commons/math/random/mod.rs:131]: ah yes got it, I'll have a follow up issue !

Comment (IceTDrinker) [reply tfhe/src/core_crypto/commons/math/random/mod.rs:131]: https://github.com/zama-ai/tfhe-rs-internal/issues/485

**Review by mayeul-zama — APPROVED** [review]
Thanks

**Pull request merged**